lib/order.rb

```diff
@@ -0,0 +1,27 @@
+class Order
+
+  attr_reader :basket, :total, :menu
+
+  def initialize
+    @basket = Hash.new(0)
+    @total = 0
+    @menu = Menu.new
+  end
+
+  def add_to_basket(item, quantity)
+    raise "Item not in menu" unless menu.items.include?(item)
+    basket[item] += quantity
+    "#{quantity} x #{item} added to your basket"
+  end
+
+  def order_summary
+    summary = basket.map { | item, quantity |
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)